app/helpers/claims_helper.rb

```diff
@@ -27,12 +27,12 @@ def claim_answers(claim)
 
   def identity_answers(claim)
     [
-      ["Full name", claim.full_name, "full-name"],
-      ["Address", claim.address, "address"],
-      ["Date of birth", l(claim.date_of_birth), "date-of-birth"],
-      ["Teacher reference number", claim.teacher_reference_number, "teacher-reference-number"],
-      ["National Insurance number", claim.national_insurance_number, "national-insurance-number"],
-      ["Email address", claim.email_address, "email-address"],
+      ["What's your full name?", claim.full_name, "full-name"],
+      ["What's your address?", claim.address, "address"],
+      ["What's your date of birth?", l(claim.date_of_birth), "date-of-birth"],
+      ["What's your teacher reference number?", claim.teacher_reference_number, "teacher-reference-number"],
+      ["What's your National Insurance number?", claim.national_insurance_number, "national-insurance-number"],
+      ["What's your email address?", claim.email_address, "email-address"],
       ["Account number", claim.bank_account_number, "bank-details"],
       ["Sort code", claim.bank_sort_code, "bank-details"],
```

Comment:
Should we stick "Bank" on the front of these for context? i.e. "Bank account number" and "Bank sort code".

Reply:
Yeah that would make it clearer, I will get on with that 😄